packages/core/src/login/ILoginOptions.ts

```diff
@@ -53,4 +53,6 @@ export default interface ILoginOptions extends ILoginInputOptions {
    * Event emitter enabling calling user-specified callbacks.
    */
   eventEmitter?: EventEmitter;
+
+  keepAlive?: boolean;
```

Comment:
`ILoginOptions` is an internal type, this is not exposed on the `Session::login` function. `keepAlive` is only set when constructing the `Session` object.